aws/data_source_aws_guardduty_detector.go

```diff
@@ -0,0 +1,95 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/guardduty"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func dataSourceAwsGuarddutyDetector() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceAwsGuarddutyDetectorRead,
+
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"enabled": {
+				Type:     schema.TypeBool,
```

Comment:
As noted below, we should probably just return a status TypeString attribute. 👍

Reply:
This has been removed, the status is now returned by the data-source.